### ISSUE TITLE
KDE Desktop Missing OpenFlight Desktop Background

### ISSUE THREAD

**Issue by ColonelPanicks** [openflighthpc/flight-desktop#6]
The KDE desktop uses a default CentOS 7 background instead of the included OpenFlight one. Fix this.

**Comment by ColonelPanicks** [comment]
This still seems to be problematic, with the new script to set the background it seems to fail (on arm, at least).

At first, the default kde background was being used, however, I attempted increasing the sleep commands between the kde desktop setup stuff in `session.sh` and, instead, ended up with just a black screen.

What I'm seeing from the session log:
```
Xvnc TigerVNC 1.8.0 - built Aug  9 2019 02:51:08
Copyright (C) 1999-2017 TigerVNC Team and many others (see README.txt)
See http://www.tigervnc.org for information on TigerVNC.
Underlying X server release 12001000, The X.Org Foundation


Wed Dec 11 15:34:41 2019
 vncext:      VNC extension running!
 vncext:      Listening for VNC connections on all interface(s), port 5902
 vncext:      created VNC server for screen 0
Sleeping [1/3]...
startkde: Starting up...

Wed Dec 11 15:34:44 2019
 Connections: accepted: 54.74.246.248::37038
 Connections: closed: 54.74.246.248::37038 (Clean disconnection)
 EncodeManager: Framebuffer updates: 0
 EncodeManager:   Total: 0 rects, 0 pixels
 EncodeManager:          0 B (1:nan ratio)
 ComparingUpdateTracker: 0 pixels in / 0 pixels out
 ComparingUpdateTracker: (1:nan ratio)
Connecting to deprecated signal QDBusConnectionInterface::serviceOwnerChanged(QString,QString,QString)
kbuildsycoca4 running...
kded(10311) LayoutMemoryPersister::restoreFromFile: Failed to open layout memory xml file for reading "/home/flight/.kde/share/apps/kded/session/keyboard/layout_memory.xml" error: 5
No outputs have backlight property
QDBusConnection: name 'org.kde.kglobalaccel' had owner '' but we thought it was ':1.5'
kded(10311) Output::readEdidData: Unable to get EDID for output "VNC-0"
Module  "resources" is registered
Module  "activities" is registered
Patching /home/flight/.kde/share/config/plasma-desktop-appletsrc
Traceback (most recent call last):
  File "/opt/flight/opt/flight-desktop/etc/types/kde/set_kde_wallpaper.py", line 36, in <module>
    'org.kde.ActivityManager', '/ActivityManager/Activities')
  File "/usr/lib64/python2.7/site-packages/dbus/bus.py", line 241, in get_object
    follow_name_owner_changes=follow_name_owner_changes)
  File "/usr/lib64/python2.7/site-packages/dbus/proxies.py", line 248, in __init__
    self._named_service = conn.activate_name_owner(bus_name)
  File "/usr/lib64/python2.7/site-packages/dbus/bus.py", line 180, in activate_name_owner
    self.start_service_by_name(bus_name)
  File "/usr/lib64/python2.7/site-packages/dbus/bus.py", line 278, in start_service_by_name
    'su', (bus_name, flags)))
  File "/usr/lib64/python2.7/site-packages/dbus/connection.py", line 651, in call_blocking
    message, timeout)
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.ServiceUnknown: The name org.kde.ActivityManager was not provided by any .service files
Sleeping [2/3]...
kquitapp(10351): "Application plasma-desktop could not be found using service org.kde.plasma-desktop and path /MainApplication."
Sleeping [3/3]...
kstart(10356) main: Omitting both --window and --windowclass arguments is not recommended
QDBusObjectPath: invalid path ""
plasma-desktop(10361): ""geometry" - conversion of "58,2,0,25" to QRectF failed"
plasma-desktop(10361): ""geometry" - conversion of "58,2,0,25" to QRectF failed"
plasma-desktop(10361): ""geometry" - conversion of "58,2,0,25" to QRectF failed"
plasma-desktop(10361): ""geometry" - conversion of "58,2,0,25" to QRectF failed"
QDBusObjectPath: invalid path ""
kded(10311) ColorD::addOutput: Failed to register device: "Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken."
kded(10311) ProfileUtils::createIccProfile: Failed to create ICC profile on cmsCreateRGBProfile
kded(10311) Output::readEdidData: Unable to get EDID for output "VNC-0"
Object::connect: No such signal QDBusAbstractInterface::subSessionOpened()
Object::connect: No such signal QDBusAbstractInterface::subSessionClosed()
Object::connect: No such signal QDBusAbstractInterface::subSessionCloseCanceled()
Object::connect: No such signal QDBusAbstractInterface::ActiveChanged(bool)
QDBusObjectPath: invalid path ""
kded(10311) ColorD::addOutput: Failed to register device: "Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken."
kded(10311) Output::readEdidData: Unable to get EDID for output "VNC-0"
Module  "features" is registered
plasma-desktop(10361)/libplasma Plasma::isPluginVersionCompatible: unversioned plugin detected, may result in instability
These are the disabled plugins: ("activitymanager_plugin_dummy", "activitymanager_plugin_nepomuk")
Initializing plugin: "activitymanager_plugin_globalshortcuts"
This is the current thread id for Activities 281473362701328 QThread(0xedff700)
plasma-desktop(10361)/libplasma Plasma::isPluginVersionCompatible: unversioned plugin detected, may result in instability
plasma-desktop(10361)/libplasma Plasma::isPluginVersionCompatible: unversioned plugin detected, may result in instability
Initializing plugin: "activitymanager_plugin_slc"
Initializing plugin: "activitymanager_plugin_virtualdesktopswitch"
VirtualDesktopSwitch::init
Module  "org.kde.ActivityManager.VirtualDesktopSwitch" is registered
Initializing plugin: "activitymanager_plugin_activityranking"
This is the current thread id for Activities 281473195765840 QThread(0xefcd180)
running in thread ActivityRanking
plasma-desktop(10361): ""geometry" - conversion of "58,2,0,25" to QRectF failed"
Initializing plugin: "activitymanager_plugin_sqlite"
Module  "org.kde.ActivityManager.Resources.Scoring" is registered
file:///usr/share/kde4/apps/plasma/packages/org.kde.plasma.tasks/contents/ui/GroupDialog.qml:44:9: QML TaskList: Cannot specify anchors for items inside Flow
This is the current thread id for Activities 281473195765840 QThread(0xefcd180)
QCoreApplication::sendPostedEvents: Cannot send posted events for objects in another thread
QCoreApplication::sendPostedEvents: Cannot send posted events for objects in another thread
QCoreApplication::sendPostedEvents: Cannot send posted events for objects in another thread
QCoreApplication::sendPostedEvents: Cannot send posted events for objects in another thread

Wed Dec 11 15:35:37 2019
 Connections: accepted: 213.83.69.6::65151
 SConnection: Client needs protocol version 3.3
 VNCSConnST:  Server default pixel format depth 24 (32bpp) little-endian rgb888
 VNCSConnST:  Client pixel format depth 32 (32bpp) little-endian rgb max
              255,255,255 shift 16,8,0
QDBusObjectPath: invalid path ""
kded(10311) ColorD::addOutput: Failed to register device: "Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken."
kded(10311) ProfileUtils::createIccProfile: Failed to create ICC profile on cmsCreateRGBProfile
kded(10311) ProfileUtils::createIccProfile: Failed to create ICC profile on cmsCreateRGBProfile
kded(10311) PowerDevil::Core::loadProfile: The profile  "AC" tried to activate "DimDisplay" a non existent action. This is usually due to an installation problem or to a configuration problem. or simlpy the action is not supported
Connecting to deprecated signal QDBusConnectionInterface::serviceOwnerChanged(QString,QString,QString)
trying to alter state of unknown activity!!
trying to alter state of unknown activity!!
VirtualDesktopSwitch::currentActivityChanged
QFileSystemWatcher: failed to add paths: /home/flight/.kde/share/config/activitymanager-pluginsrc
plasma-desktop(10361)/libplasma Plasma::isPluginVersionCompatible: unversioned plugin detected, may result in instability
plasma-desktop(10361)/libplasma Plasma::isPluginVersionCompatible: unversioned plugin detected, may result in instability
plasma-desktop(10361)/libplasma Plasma::isPluginVersionCompatible: unversioned plugin detected, may result in instability
========================>  0 0
file:///usr/share/kde4/apps/plasma/packages/org.kde.systemtray/contents/ui/IconsList.qml:42: TypeError: Result of expression 'delegate_highlight.marginHints' [undefined] is not an object.
static bool QDeclarativeMetaType::isModule(const QByteArray&, int, int) Qt 4.7 import detected; please note that Qt 4.7 is directly reusable as QtQuick 1.x with no code changes. Continuing, but startup time will be slower.
trying to alter state of unknown activity!!
file:///usr/lib64/kde4/imports/org/kde/plasma/components/SectionScroller.qml:103: TypeError: Result of expression 'listView' [null] is not an object.
file:///usr/lib64/kde4/imports/org/kde/plasma/components/SectionScroller.qml:175: ReferenceError: Can't find variable: sectionsRepeater
file:///usr/lib64/kde4/imports/org/kde/plasma/components/SectionScroller.qml:103: Error: Cannot assign [undefined] to QString
file:///usr/lib64/kde4/imports/org/kde/plasma/extras/ScrollArea.qml:120: TypeError: Result of expression 'horizontalScrollBar' [null] is not an object.
file:///usr/lib64/kde4/imports/org/kde/plasma/extras/ScrollArea.qml:120: TypeError: Result of expression 'horizontalScrollBar' [null] is not an object.
file:///usr/share/kde4/apps/plasma/plasmoids/notifier/contents/ui/devicenotifier.qml:224:13: QML QDeclarativeListView_QML_76: Possible anchor loop detected on fill.
file:///usr/share/kde4/apps/plasma/plasmoids/notifier/contents/ui/devicenotifier.qml:224:13: QML QDeclarativeListView_QML_76: Possible anchor loop detected on fill.
file:///usr/share/kde4/apps/plasma/plasmoids/notifier/contents/ui/devicenotifier.qml:224:13: QML QDeclarativeListView_QML_76: Possible anchor loop detected on fill.
file:///usr/share/kde4/apps/plasma/plasmoids/notifier/contents/ui/devicenotifier.qml:224:13: QML QDeclarativeListView_QML_76: Possible anchor loop detected on fill.
file:///usr/share/kde4/apps/plasma/plasmoids/notifier/contents/ui/devicenotifier.qml:224:13: QML QDeclarativeListView_QML_76: Possible anchor loop detected on fill.
file:///usr/share/kde4/apps/plasma/plasmoids/notifier/contents/ui/devicenotifier.qml:224:13: QML QDeclarativeListView_QML_76: Possible anchor loop detected on fill.
file:///usr/share/kde4/apps/plasma/plasmoids/notifier/contents/ui/devicenotifier.qml:224:13: QML QDeclarativeListView_QML_76: Possible anchor loop detected on fill.
Couldn't find synaptics properties. No synaptics driver loaded?
Object::connect: No such signal QDBusAbstractInterface::StateChanged(uint)
SELinux Troubleshooter: Applet requires SELinux be enabled to run.
QDBusConnection for control created "/Mixers/PulseAudio__Playback_Devices_1/auto_null"
New PolkitAgentListener  0x325f9a20
Adding new listener  PolkitQt1::Agent::Listener(0x327a1100) for  0x325f9a20

** (process:10453): WARNING **: 15:36:27.522: Unable to register authentication agent: GDBus.Error:org.freedesktop.PolicyKit1.Error.Failed: An authentication agent already exists for the given subject
"Cannot register authentication agent: GDBus.Error:org.freedesktop.PolicyKit1.Error.Failed: An authentication agent already exists for the given subject"
Connecting to deprecated signal QDBusConnectionInterface::serviceOwnerChanged(QString,QString,QString)
kded(10311)/kdecore (trader) KServiceTypeTrader::defaultOffers: KServiceTypeTrader: serviceType  "SolidModemManager"  not found
kded(10311)/kmix sink_input_cb: Ignoring sink-input due to it being designated as an event and thus handled by the Event slider
QDBusConnection for control created "/Mixers/PulseAudio__Playback_Streams_1/restore_sink_input_by_media_role_event"
Object::connect: No such slot PrintersEngine::printerRemoved(QString)
Object::connect:  (receiver name: 'Printers Engine')
klipper(10472) Klipper::loadHistory: Failed to load history resource. Clipboard history cannot be read. :  History file does not exist
kmix(10488) sink_input_cb: Ignoring sink-input due to it being designated as an event and thus handled by the Event slider
QDBusConnection for control created "/Mixers/PulseAudio__Playback_Devices_1/auto_null"
kmix(10488) sink_input_cb: Ignoring sink-input due to it being designated as an event and thus handled by the Event slider
QDBusConnection for control created "/Mixers/PulseAudio__Playback_Streams_1/restore_sink_input_by_media_role_event"
plasma-desktop(10361)/libakonadi Akonadi::AgentManagerPrivate::createDBusInterface: AgentManager failed to get a valid AgentManager DBus interface. Error is: 1 "org.freedesktop.DBus.Error.NameHasNoOwner" "Could not get owner of name 'org.freedesktop.Akonadi.Control': no such name"
plasma-desktop(10361)/libakonadi Akonadi::SessionPrivate::socketError: Socket error occurred: "QLocalSocket::connectToServer: Invalid name"
plasma-desktop(10361)/libakonadi Akonadi::SessionPrivate::socketError: Socket error occurred: "QLocalSocket::connectToServer: Invalid name"
Connecting to deprecated signal QDBusConnectionInterface::serviceOwnerChanged(QString,QString,QString)
search paths:  ("/usr/local/bin", "/usr/bin", "/usr/local/sbin", "/usr/sbin", "/home/flight/.local/bin", "/home/flight/bin", "/usr/sbin", "/usr/local/sbin", "/usr/local/libexec", "/usr/libexec", "/opt/mysql/libexec", "/opt/local/lib/mysql5/bin", "/opt/mysql/sbin")
plasma-desktop(10361)/libplasma Plasma::PackagePrivate::isValid: Could not find required directory images
plasma-desktop(10361)/plasma StatusNotifierItemSource::refreshCallback: DBusMenu disabled for this application
akonadi.collectionattributetable                   OK
akonadi.collectionmimetyperelation                 OK
akonadi.collectionpimitemrelation                  OK
akonadi.collectiontable                            OK
akonadi.flagtable                                  OK
akonadi.mimetypetable                              OK
akonadi.parttable                                  OK
akonadi.pimitemflagrelation                        OK
akonadi.pimitemtable                               OK
akonadi.resourcetable                              OK
akonadi.schemaversiontable                         OK
mysql.columns_priv                                 OK
mysql.db                                           OK
mysql.event                                        OK
mysql.func                                         OK
mysql.help_category                                OK
mysql.help_keyword                                 OK
mysql.help_relation                                OK
mysql.help_topic                                   OK
mysql.host                                         OK
mysql.ndb_binlog_index                             OK
mysql.plugin                                       OK
mysql.proc                                         OK
mysql.procs_priv                                   OK
mysql.proxies_priv                                 OK
mysql.servers                                      OK
mysql.tables_priv                                  OK
mysql.time_zone                                    OK
mysql.time_zone_leap_second                        OK
mysql.time_zone_name                               OK
mysql.time_zone_transition                         OK
mysql.time_zone_transition_type                    OK
mysql.user                                         OK
Nepomuk Query Server not available
Connecting to deprecated signal QDBusConnectionInterface::serviceOwnerChanged(QString,QString,QString)
search paths:  ("/usr/local/bin", "/usr/bin", "/usr/local/sbin", "/usr/sbin", "/home/flight/.local/bin", "/home/flight/bin")
search paths:  ("/usr/lib64", "/usr/lib64/qt4/plugins/", "/usr/lib64/kde4/", "/usr/lib64/kde4/plugins/", "/usr/lib/qt4/plugins/", "/usr/lib64/qt4/plugins", "/usr/lib64/kde4/plugins", "/usr/bin", "/home/flight/.kde/lib64/kde4/")
search paths:  ("/usr/lib64", "/usr/lib64/qt4/plugins/", "/usr/lib64/kde4/", "/usr/lib64/kde4/plugins/", "/usr/lib/qt4/plugins/", "/usr/lib64/qt4/plugins", "/usr/lib64/kde4/plugins", "/usr/bin", "/home/flight/.kde/lib64/kde4/")
search paths:  ("/usr/lib64", "/usr/lib64/qt4/plugins/", "/usr/lib64/kde4/", "/usr/lib64/kde4/plugins/", "/usr/lib/qt4/plugins/", "/usr/lib64/qt4/plugins", "/usr/lib64/kde4/plugins", "/usr/bin", "/home/flight/.kde/lib64/kde4/")
search paths:  ("/usr/lib64", "/usr/lib64/qt4/plugins/", "/usr/lib64/kde4/", "/usr/lib64/kde4/plugins/", "/usr/lib/qt4/plugins/", "/usr/lib64/qt4/plugins", "/usr/lib64/kde4/plugins", "/usr/bin", "/home/flight/.kde/lib64/kde4/")
kmix(10488) sink_input_cb: Ignoring sink-input due to it being designated as an event and thus handled by the Event slider
kded(10311)/kmix sink_input_cb: Ignoring sink-input due to it being designated as an event and thus handled by the Event slider
```

**Comment by ColonelPanicks** [comment]
Having tested again this is no longer an issue.
